src/main/scala/chisel3/iotesters/Driver.scala

```diff
@@ -121,32 +138,45 @@ object Driver {
   }
   /**
     * Runs the ClassicTester and returns a Boolean indicating test success or failure
-    * @@backendType determines whether the ClassicTester uses verilator or the firrtl interpreter to simulate the circuit
-    * Will do intermediate compliation steps to setup the backend specified, including cpp compilation for the verilator backend and firrtl IR compilation for the firrlt backend
+    * @@backendType determines whether the ClassicTester uses verilator or the firrtl interpreter to simulate
+    * the circuit.
+    * Will do intermediate compliation steps to setup the backend specified, including cpp compilation for the
+    * verilator backend and firrtl IR compilation for the firrlt backend
+    *
+    * This apply method is a convenient short form of the [[Driver.execute()]] which has many more options
+    *
+    * The following tests a chisel CircuitX with a CircuitXTester setting the random number seed to a fixed value and
+    * turning on verbose tester output.  The result of the overall test is put in testsPassed
+    *
+    * @example {{{
+    *           val testsPassed = iotesters.Driver(() => new CircuitX, testerSeed = 0L, verbose = true) { circuitX =>
+    *             CircuitXTester(circuitX)
+    *           }
+    * }}}
+    *
+
+    * @param dutGen      This is the device under test.
+    * @param backendType The default backend is "firrtl" which uses the firrtl interperter. Other options
+    *                    "verilator" will use the verilator c++ simulation generator
+    *                    "vcs" will use the VCS simulation
+    * @param verbose     Setting this to true will make the tester display information on peeks,
+    *                    pokes, steps, and expects.  By default only failed expects will be printed
+    * @param testerSeed  Set the random number generator seed
+    * @param testerGen   This is a test harness subclassing PeekPokeTester for dutGen,
+    * @return            This will be true if all tests in the testerGen pass
     */
-  def apply[T <: Module](dutGen: () => T, backendType: String = "firrtl")(
+  def apply[T <: Module](
+      dutGen: () => T,
+      backendType: String = "firrtl",
```

Comment:
Decide not do this for now.  It would be just another different hidden place to control backend and is not documented as far as we know.